libursa/src/bn/openssl.rs

```diff
@@ -6,15 +6,16 @@ use openssl::bn::{BigNum, BigNumContext, BigNumRef, MsbOption};
 use openssl::error::ErrorStack;
 use openssl::hash::{hash, Hasher, MessageDigest};
 
-#[cfg(feature = "serialization")]
-use serde::ser::{Error as SError, Serialize, Serializer};
-
-#[cfg(feature = "serialization")]
-use serde::de::{Deserialize, Deserializer, Error as DError, Visitor};
+#[cfg(feature = "serde")]
+use serde::{de::Visitor, Deserialize, Deserializer, Serialize, Serializer};
+//use serde::ser::{Error as SError, Serialize, Serializer};
```

Comment:
Is the commented-out code here intentional?

Reply:
I just need to delete the commented code.